### PR TITLE
Version 5.6.0 update release notes

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

**Review by ebfull — APPROVED** [review]

File: doc/release-notes.md [hunk 1]
```diff
@@ -25,18 +25,46 @@ is not possible to compute that without knowing the number of logical actions
...
+caller precomputed node values within the Sapling and Orchard note commitment 
+trees. Wallets can make use of these precomputed values to make their existing
+notes spendable without needing to fully scan the sub-trees whose roots
+correspond to the returned node values.
+
+In conjunction with this change, the `getblock` RPC method now returns an
+additional `trees` field as part of its result. The value for this field is an
+object that contains the final sizes of the Sapling and Orchard note commitment
+trees after the block's note commitments have been appended to their respective
+trees.
```
Comment (daira) [inline doc/release-notes.md:43]: Are these only present when `-experimentalfeatures=1` and `-lightwalletd=1` are set?

Comment (ebfull) [reply doc/release-notes.md:43]: I did not gate this on lightwalletd being enabled.

Comment (ebfull) [reply doc/release-notes.md:43]: I don't think there's a reason to condition this on lightwalletd info either. This is good for debugging even outside the lightwalletd context.

**Review by daira — COMMENTED** [review]
Post-hoc ACK.

**Pull request merged**